doc/_admin-guide/060_Sources/031_Hypr/000_Hypr_options.md

```diff
@@ -0,0 +1,68 @@
+---
+title: `hypr-audit-trail()` and `hypr-app-audit-trail()` source options
+id: adm-src-hypr-opt
+---
+
+The `hypr-audit-trail()` and `hypr-app-audit-trail() sources have the following options:
+
+## url()
+
+|  Type:   |      url|
+|  Default:|         |
+
+*Description:* A custom URL for Hypr API access ("https://<custom domain>.hypr.com")
```

Comment:
gt / lt characters are not escaped, so custom domain part is not displayed bc parsed as a html tag.

```suggestion
*Description:* A custom URL for Hypr API access ("https://\<custom domain\>.hypr.com")
```

Reply:
can't we use some textual reference and a normal external link instead?